aries_vcx/src/handlers/proof_presentation/verifier/states/presentation_proposal_received.rs

```diff
@@ -1,10 +1,10 @@
 use crate::messages::proof_presentation::presentation_proposal::PresentationProposal;
-use crate::messages::proof_presentation::presentation_request::PresentationRequestData;
+use crate::messages::proof_presentation::presentation_request::PresentationRequest;
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Default)]
 pub struct PresentationProposalReceivedState {
     pub presentation_proposal: PresentationProposal,
-    pub presentation_request_data: Option<PresentationRequestData>,
+    pub presentation_request: Option<PresentationRequest>,
```

Comment:
Now we store `PresentationRequest` itself, as opposed to `PresentationRequestData` which was **input** for indy function generating `PresentationRequest`